db/migrations/2023080311350006-add-comments-github-to-comments-table.js

```diff
@@ -0,0 +1,165 @@
+const BOT_NAME = 'bepro-bot'
+
+async function handleAddComments(comment, id, type, prId){
+  const regexGithubLogin = /@(\w+)/;
+  const bodyGithubLogin = regexGithubLogin.exec(comment.body);
+
+  const getCommentCreateData = (userId, userAddress, body) => ({
+    userId,
+    userAddress,
+    comment: body,
+    issueId: id,
+    hidden: false,
+    type,
+    created_at: comment.created_at,
+    updated_at: comment.updated_at,
+    ... prId ? { deliverableId: prId } : null
+  }) 
+
+  const getUser = (name) => UserModel.findOne({
+    where: {
+      githubLogin: name
+    }
+  })
+
+  if(bodyGithubLogin && comment.user.login === BOT_NAME){
+    const user = await getUser(bodyGithubLogin[1])
+    let text;
+
+    if(user){
+      //Example text: "@name  is working on this."
+      if(/working/.test(comment.body) && type === 'issue') text = `i'm working on this.`
+      //Example text: "has a solution - [check your bounty](https://)"
+      if(/solution/.test(comment.body) && type === 'issue') {
+        const regexSolution = /(?<=- ).*/
+        text = `finished a solution - ${comment.body.match(regexSolution)[0]}`
+      }
+      //Example text: "reviewed this Pull Request with the following message:    Message"
+      if(/Pull Request/.test(comment.body) && type === 'deliverable') {
+        const regexPr = /(?<=:\s*).*/;
+        text = comment.body.match(regexPr)[0]
+      }
+
+      await CommentsModel.create(getCommentCreateData(user.id, user.address, text))
+    }
+  } else {
+    const user = await getUser(comment.user.login)
+
+    if(user)
+      await CommentsModel.create(getCommentCreateData(user.id, user.address, comment.body))
+  }
+}
+
+async function up(queryInterface, Sequelize) {
+  if (SKIP_MIGRATION_SEED_COMMENTS_DATE_GITHUB === "true") return;
+
+  [
+    ChainModel,
+    NetworkModel,
+    IssueModel,
+    CuratorsModel,
+    RepositoryModel,
+    PullRequestModel,
+    MergeProposalModel,
+    TokenModel,
+    BenefactorModel,
+    DisputeModel,
+    UserPaymentsModel,
+    DeveloperModel,
+    CommentsModel,
+    UserModel
+  ].forEach((model) => model.init(queryInterface.sequelize));
+
+  [ChainModel, NetworkModel, IssueModel].forEach((model) =>
+    model.associate(queryInterface.sequelize.models)
+  );
+
+  const chains = await ChainModel.findAll({
+    where: {
+      registryAddress: {
+        [Op.ne]: null,
+      },
+    },
+    include: [
+      {
+        association: "networks",
+        required: true,
+        include: [
+          {
+            association: "issues",
+            where: {
+              state: { [Op.not]: "pending" },
+            },
+            include: [{ association: "repository" }, {association: "pullRequests"}],
+            required: true,
+          },
+        ],
+      },
+    ],
+  });
+
+  if (!chains.length) return;
+
+  try {
+    for (const { networks } of chains) {
+
+      for (const { issues } of networks) {
+
+        for (const issue of issues) {
```

Comment:
we don't need to iterate through all the chains, and then through the networks, and finally issues; we can simply iterate on the issues by using `IssueModel.findAll()`